build.gradle

```diff
@@ -1,14 +1,16 @@
 //
 plugins {
-    id 'org.octopusden.octopus-release-management'
     id 'groovy'
     id 'java-gradle-plugin'
+    id 'io.github.gradle-nexus.publish-plugin'
+    id 'signing'
 }
 
 group = 'org.octopusden.octopus'
 description "License gradle plugin"
 
 repositories {
+    mavenLocal()
```

Comment:
@adellinekania why do we need mavenLocal() here?

Reply:
@adellinekania please check https://docs.gradle.org/current/userguide/declaring_repositories_adv.html#sec:case-for-maven-local

Therefore my question could be clarified:
- do the project contains dependencies that might be build locally for testing purposes?
- if it does, don't you use init.gradle that already specifies mavenLocal()?

mavenLocal() definetely do not required to build the project on GitHub so why should we explicitly define it in source code?

Reply:
Yes, after reviewing it, I realize that `mavenLocal` is only needed for the functional tests to use the built plugin specified in `init.gradle`. I'll remove it. Thank you!